.changeset/tricky-beds-kiss.md

```diff
@@ -0,0 +1,19 @@
+---
+"@latticexyz/cli": patch
+"@latticexyz/store": patch
+"@latticexyz/world": patch
```

Comment:
should we add another changeset for https://github.com/latticexyz/mud/pull/1385?

Reply:
ohh true forgot it there. Could do that in a separate followup though, this PR doesn't actually use the `WorldFactory` yet (because calling `initialize` in a separate transaction is supported it's not strictly necessary.)